docs/notes.rst

```diff
@@ -1,8 +1,42 @@
-Version 1.8.1
+Version 1.8.2
 =============
 
 This release includes the following features:
 
+* Modified unknown writers default behavior.
+* Multiple initial PDP announcements.
+* Flag to avoid builtin multicast.
+* *STRICT_REALTIME* compilation flag.
+
+It also adds the following bug fixes and improvements:
+
+* Fix for setting `nullptr` in a fixed string.
+* Fix for not sending GAP in several cases.
+* Solve *Coverity* report issues.
+* Fix issue of *fastrtpsgen* failing to open *IDL.g4* file.
+* Fix unnamed lock in *AESGCMGMAC_KeyFactory.cpp*.
+* Improve *XMLProfiles* example.
+* Multicast is now sent through *localhost* too.
+* *BitmapRange* now implements sliding window.
```

Comment:
Can you confirm that the following:
- Fix issue of *fastrtpsgen* failing to open *IDL.g4* file
- Improve *XMLProfiles* example
- Multicast is now sent through localhost too
- BitmapRange now implements sliding window

are also part of 1.8.2, please?

Reply:
> Can you confirm that the following:
> 
> * Fix issue of _fastrtpsgen_ failing to open _IDL.g4_ file
cfaf5307dae3ab94e6ff929f7507779585895691
> * Improve _XMLProfiles_ example
519b30ed4856c38943ee69c602224d3ea0a5e374
> * Multicast is now sent through localhost too
b45aaa41ca2427be25e0dc7f4e9756fc8a6fc37d
> * BitmapRange now implements sliding window
c447a38a2432ffeca3cda6d09e9022278f97f7e0
> 
> are also part of 1.8.2, please?

Reply:
Thanks!